### DISCUSSION TITLE
Heure de fin de la phase d'anonymisation

### DISCUSSION THREAD

**Discussion by jaalmoes** [beng-git/DARC2022#8] [Q&A]
Bonjour,

On avait dit que le rendu devait être fait le 13 (aujourd'hui) mais on n'avait pas précisié d'heure :)

**Comment by nancy-awad** [comment]
Nous ne fermerons pas la phase d'anonymisation avant minuit.
Cependant, nous sommes prêts à prolonger la phase d'anonymisation pour une courte durée (quelques heures ) si vous en avez besoin.

**Reply by jaalmoes** [reply]
Super merci ! Il est possible qu'on en ait besoin en effet

**Reply by jaalmoes** [reply]
Oui il va bien falloir quelques heures en plus pour terminer de notre coté
